websites/G/Gemini/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "1145729748803272815",
+		"name": "mallusrgreat"
+	},
+	"service": "Gemini",
+	"description": {
+		"en": "Gemini gives you direct access to Google AI. Get help with writing, planning, learning, and more."
+	},
+	"url": "gemini.google.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/2wXchS8.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be 512x512px